### ISSUE TITLE
Argument for @NotNull parameter 'psiElement'

### ISSUE THREAD

**Issue by Krazitchek** [Haehnchen/idea-php-symfony2-plugin#653]
Recurrent error since 0.11.105 release:

Argument for @NotNull parameter 'psiElement' of fr/adrienbrault/idea/symfony2plugin/form/util/FormUtil.getFormTypeClassOnParameter must not be null
java.lang.IllegalArgumentException: Argument for @NotNull parameter 'psiElement' of fr/adrienbrault/idea/symfony2plugin/form/util/FormUtil.getFormTypeClassOnParameter must not be null
	at fr.adrienbrault.idea.symfony2plugin.form.util.FormUtil.getFormTypeClassOnParameter(FormUtil.java)
	at fr.adrienbrault.idea.symfony2plugin.form.util.FormUtil.getFormTypeClass(FormUtil.java:146)
	at fr.adrienbrault.idea.symfony2plugin.form.util.FormUtil.resolveFormGetterCall(FormUtil.java:129)
	at fr.adrienbrault.idea.symfony2plugin.form.util.FormUtil.resolveFormGetterCallMethod(FormUtil.java:155)
	at fr.adrienbrault.idea.symfony2plugin.form.FormTypeReferenceContributor$5.getReferencesByElement(FormTypeReferenceContributor.java:261)
	at com.intellij.psi.impl.source.resolve.reference.ReferenceProvidersRegistryImpl.doGetReferencesFromProviders(ReferenceProvidersRegistryImpl.java:120)
	at com.intellij.psi.impl.source.resolve.reference.ReferenceProvidersRegistry.getReferencesFromProviders(ReferenceProvidersRegistry.java:66)
	at com.intellij.psi.impl.source.resolve.reference.ReferenceProvidersRegistry.getReferencesFromProviders(ReferenceProvidersRegistry.java:60)
	at com.jetbrains.php.lang.psi.elements.impl.StringLiteralExpressionImpl.getReferences(StringLiteralExpressionImpl.java:283)
	at org.intellij.plugins.intelliLang.references.InjectedReferencesContributor.getInjectedReferences(InjectedReferencesContributor.java:56)
	at org.intellij.plugins.intelliLang.references.InjectedReferencesInspection$1.visitElement(InjectedReferencesInspection.java:38)
	at com.jetbrains.php.lang.psi.elements.impl.StringLiteralExpressionImpl.accept(StringLiteralExpressionImpl.java:29)
	at com.intellij.codeInspection.InspectionEngine.acceptElements(InspectionEngine.java:81)
	at com.intellij.codeInsight.daemon.impl.LocalInspectionsPass$4.process(LocalInspectionsPass.java:320)
	at com.intellij.codeInsight.daemon.impl.LocalInspectionsPass$4.process(LocalInspectionsPass.java:315)
	at com.intellij.concurrency.ApplierCompleter.b(ApplierCompleter.java:122)
	at com.intellij.concurrency.ApplierCompleter.a(ApplierCompleter.java:211)
	at com.intellij.concurrency.ApplierCompleter.b(ApplierCompleter.java:140)
	at com.intellij.concurrency.ApplierCompleter.access$000(ApplierCompleter.java:44)
	at com.intellij.concurrency.ApplierCompleter$1.run(ApplierCompleter.java:85)
	at com.intellij.openapi.application.impl.ApplicationImpl.tryRunReadAction(ApplicationImpl.java:1169)
	at com.intellij.concurrency.ApplierCompleter$2.run(ApplierCompleter.java:94)
	at com.intellij.openapi.progress.impl.CoreProgressManager.a(CoreProgressManager.java:446)
	at com.intellij.openapi.progress.impl.CoreProgressManager.a(CoreProgressManager.java:443)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:392)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:54)
	at com.intellij.concurrency.ApplierCompleter.a(ApplierCompleter.java:106)
	at com.intellij.concurrency.ApplierCompleter.compute(ApplierCompleter.java:82)
	at jsr166e.CountedCompleter.exec(CountedCompleter.java:684)
	at jsr166e.ForkJoinTask.doExec(ForkJoinTask.java:260)
	at jsr166e.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:858)
	at jsr166e.ForkJoinPool.scan(ForkJoinPool.java:1687)
	at jsr166e.ForkJoinPool.runWorker(ForkJoinPool.java:1642)
	at jsr166e.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:108)

**Comment by falsecrypt** [comment]
I'm getting the same: IllegalArgumentException: Argument for @NotNull parameter 'psiElement' of fr/adrienbrault/idea/symfony2plugin/form/util/FormUtil.getFormTypeClassOnParameter must not be null.
It happens if i doubleclick on a string in get(), e.g. on 'location':
`$form->get('location')->...`

**Comment by tuanalumi** [comment]
I have this issue too.

This happens when you access form data like this:

```php
$form->handleRequest($request);
$location = $form->get('location')->getData(); // this will give the error message
```

```php
$location = $form->getData()['location']; // this is ok
```

**Comment by ureimers** [comment]
Thanks @tuanalumi that helps.
The files have to be open too (i.e., be actively inspected by PS). As soon as you close them, no additional Exceptions will be thrown.

**Comment by scaytrase** [comment]
have the same error

**Comment by ahoulgrave** [comment]
Same error here

11:08:51 AM IllegalArgumentException: Argument for @NotNull parameter 'psiElement' of fr/adrienbrault/idea/symfony2plugin/form/util/FormUtil.getFormTypeClassOnParameter must not be null

Over and over until:

11:31:43 AM com.intellij.diagnostic.MessagePool$TooManyErrorsException: Too many IDE fatal errors. Monitoring stopped

**Comment by tuanalumi** [comment]
Guys, this seems to be fixed in 0.11.106.

> Check null before calling getFormTypeClassOnParameter in FormUtil #650
